packages/next/src/lib/metadata/get-metadata-route.ts

```diff
@@ -20,6 +20,9 @@ export function normalizeMetadataRoute(page: string) {
     if (route === '/robots') {
       route += '.txt'
     }
+    if (route === '/manifest') {
+      route += '.json'
```

Comment:
It appears that `.webmanifest` is registered with IANA, not `.json`.

https://w3c.github.io/manifest/#iana-considerations

Reply:
It also works with json extension, but will change to webmanifest as it's the closet one to the spec. thanks
> Developers can also choose a different extension (e.g. .json) or none at all (e.g. /api/GetManifest), but are encouraged to transfer the manifest using the [application/manifest+json](https://w3c.github.io/manifest/#dfn-application-manifest-json) [MIME type](https://mimesniff.spec.whatwg.org/#mime-type), although any [JSON MIME type](https://mimesniff.spec.whatwg.org/#json-mime-type) is ok.